src/res/translation/translation_de_DE.ts

```diff
@@ -2461,36 +2461,46 @@
     </message>
     <message>
         <location filename="../../util.cpp" line="1110"/>
+        <source>Vocal Baritone</source>
+        <translation>Gesang Bariton</translation>
+    </message>
+    <message>
+        <location filename="../../util.cpp" line="1111"/>
+        <source>Vocal Lead</source>
+        <translation>Gesang Lead</translation>
```

Comment:
Good that you added the translations, too. But you have used words like "Lead" in the translations. I would translate it differently. The problem is that the translators will not easily see that there is an issue in the translation since only for missing translations you get a question mark in the Qt Linguist. For all translations which are marked as ready you will not easily find them. So for translations where you are not 100 % sure that they are correct, you should leave them empty instead so that the translators will find them easily.

Reply:
@corrados Thanks, so just leaving: `<translation></translation>`

Reply:
I believe Qt Linguist will only show the question mark if the _translation_ element has the attribute _type_ set to _unfinished_: `<translation type="unfinished"></translation>`

Reply:
Oh, yes. Thanks for that hint. That's right.
@AdrienDS Can you please fix that?